syntax = "proto3";

message HmsData {
  enum Error {
    LOW_BATTERY = 0;
    OFF_TRACK = 1; // robot off track
  }
  float batteryVoltage = 1;
  uint32 nCells = 2;
  string errorInfo = 3;
  // Error error = 4;
}
